packages/create-plugin/src/migrations/utils.ts

```diff
@@ -132,3 +133,174 @@ export function installNPMDependencies(context: Context) {
     execSync(installCmd, { cwd: context.basePath, stdio: 'inherit' });
   }
 }
+
+export function readJsonFile<T extends object = any>(context: Context, path: string): T {
+  if (!context.doesFileExist(path)) {
+    throw new Error(`Cannot find ${path}`);
+  }
+  try {
+    return JSON.parse(context.getFile(path) || '{}');
```

Comment:
Why do we need the `|| '{}'? If the file is empty, then we would like to throw the error (not able to parse) - or am I wrong?